### PR TITLE
render collection background

### PR DESCRIPTION
**Pull request by davodesign84**

### CODE REVIEW

File: packages/kauri-web/components/containers/Collection/View.js [hunk 1]
```diff
@@ -31,14 +31,15 @@ const HeaderContainer = styled(ContentContainer)`
...
+    console.log(background);
     return (
       <div>
-        <HeaderContainer>
+        <HeaderContainer background={background}>
           <CollectionHeader
             name={name}
             description={description}
-            updated={date_created}
+            updated={dated_updated || date_created}
```
[the remark this answers is not in the document]
Comment (davodesign84) [reply packages/kauri-web/components/containers/Collection/View.js]: uh? no, if there is no date_updated will default to date_created :P

[the remark this answers is not in the document]
Comment (rej156) [reply packages/kauri-web/components/containers/Collection/View.js]: dated_

[the remark this answers is not in the document]
Comment (davodesign84) [reply packages/kauri-web/components/containers/Collection/View.js]: ohhhh 🐰

File: packages/kauri-web/components/containers/Collection/View.js [hunk 2]
```diff
@@ -31,14 +31,15 @@ const HeaderContainer = styled(ContentContainer)`
 class CollectionPage extends Component<Props> {
   render() {
     if (!this.props.data || !this.props.data.collection) return null;
-    const { name, description, date_created, owner, sections } = this.props.data.collection
+    const { name, background, description, date_created, dated_updated, owner, sections } = this.props.data.collection
+    console.log(background);
```
Comment (rej156) [inline packages/kauri-web/components/containers/Collection/View.js]: Console.log

File: packages/kauri-web/components/containers/Collection/View.js [hunk 3]
```diff
@@ -31,14 +31,15 @@ const HeaderContainer = styled(ContentContainer)`
 class CollectionPage extends Component<Props> {
   render() {
     if (!this.props.data || !this.props.data.collection) return null;
-    const { name, description, date_created, owner, sections } = this.props.data.collection
+    const { name, background, description, date_created, dated_updated, owner, sections } = this.props.data.collection
```
Comment (rej156) [inline packages/kauri-web/components/containers/Collection/View.js]: Destructuring wrong

File: packages/kauri-web/components/containers/Collection/View.js [hunk 4]
```diff
@@ -31,15 +31,15 @@ const HeaderContainer = styled(ContentContainer)`
 class CollectionPage extends Component<Props> {
   render() {
     if (!this.props.data || !this.props.data.collection) return null;
-    const { name, background, description, date_created, dated_updated, owner, sections } = this.props.data.collection
+    const { name, background, description, date_created, date_updated, owner, sections } = this.props.data.collection
     console.log(background);
```
Comment (rej156) [inline packages/kauri-web/components/containers/Collection/View.js]: Console.log

**Pull request merged**